src/jarabe/desktop/keydialog.py

```diff
@@ -120,6 +129,12 @@ def create_security(self):
     def get_response_object(self):
         return self._response
 
+    def _toggle_visibility_cb(self, entry):
+        if self._show_pass_toggle.get_active():
```

Comment:
you don't have to use an if statment here... `self._entry.set_visibility(self._show_pass_toggle.get_active())`